src/QSymbolicsBase/express.jl

```diff
@@ -78,7 +78,17 @@ end
 ##
 
 """Representation using kets, bras, density matrices, and superoperators governed by `QuantumOptics.jl`."""
-struct QuantumOpticsRepr <: AbstractRepresentation end
+struct QuantumOpticsRepr <: AbstractRepresentation 
+    fock_cutoff::Union{Nothing, Int}
+    function QuantumOpticsRepr(fock_cutoff::Union{Nothing, Int})
```

Comment:
This should probably not be an inner constructor. I also made a bunch of breaks to the documentation that currently would not work, but basically my goal is to make this a keyword argument constructor, so that it is easy to add more parameters to it in the future.

Reply:
Sure, thats OK with me.